Makefile

```diff
@@ -23,6 +28,9 @@ define make_docker
 		--build-arg SVC=$(svc) \
 		--build-arg GOARCH=$(GOARCH) \
 		--build-arg GOARM=$(GOARM) \
+		--build-arg VERSION=$(shell git describe --abbrev=0 --tags) \
```

Comment:
Please check if you need to change our Dockerfile as well, as it is `alpine go`, maybe you need to install Git into it (i.e. try to build Docker images, see if compilation will work).

Reply:
@drasko Compilation work, I already tested. There is no need to install git in the Dockerfile since you pass this values as arguments.